pandas/core/generic.py

```diff
@@ -1892,6 +1897,34 @@ def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
             Optional specifying the datatype for columns. The SQL type should
             be a SQLAlchemy type, or a string for sqlite3 fallback connection.
 
+        Returns
+        --------
+            None
+
+        See Also
+        --------
+        pandas.read_sql_query : read a DataFrame from a table
+
+        Examples
+        --------
+        >>> from sqlalchemy import create_engine
+        >>> engine = create_engine('sqlite://', echo=False)
+        >>> df = pd.DataFrame({'name' : ['User 1', 'User 2', 'User 3']})
+        >>> # create a table from scratch with 3 rows
+        >>> df.to_sql('users', con=engine, if_exists='replace')
```

Comment:
can you add blank lines between cases
also for comments, don't use the leading '>>>'